### PR TITLE
Lon 1054

### PR DESCRIPTION
**Pull request by SandraPG85**

### CODE REVIEW

**Comment by vguzmanp** [comment]
For the future: commit messages should be capitalized

File: LongoMatch.GUI/LongoMatch.GUI.csproj [hunk 1]
```diff
@@ -223,9 +223,11 @@
 		<Reference Include="System" />
 		<Reference Include="System.Core" />
 		<Reference Include="System.Drawing" />
-		<Reference Include="Mono.Addins, Version=1.0.0.0, Culture=neutral, PublicKeyToken=0738eb9f132ed756" />
 		<Reference Include="Mono.Posix" />
 		<Reference Include="Microsoft.CSharp" />
+		<Reference Include="Mono.Addins">
+		  <HintPath>..\..\packages\Mono.Addins.1.3\lib\net40\Mono.Addins.dll</HintPath>
```
Comment (vguzmanp) [inline LongoMatch.GUI/LongoMatch.GUI.csproj]: Why is it like this now?

File: LongoMatch.Plugins.Stats/StatsUIPlugin.cs [hunk 2]
```diff
@@ -15,20 +15,18 @@
...
-[assembly:Addin]
-[assembly:AddinAuthor ("LongoMatch Project")]
-[assembly:AddinName ("Stats")]
-[assembly:AddinDescription ("Statistics plugin")]
-[assembly:AddinDependency ("LongoMatch", "1.1")]
+// [assembly: Addin]
+// [assembly: AddinAuthor ("LongoMatch Project")]
+// [assembly: AddinName ("Stats")]
+// [assembly: AddinDescription ("Statistics plugin")]
+// [assembly: AddinDependency ("LongoMatch", "1.1")]
```
Comment (vguzmanp) [inline LongoMatch.Plugins.Stats/StatsUIPlugin.cs]: Commented code

File: LongoMatch.Plugins.GStreamer/packages.config [hunk 3]
```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Mono.Addins" version="1.3" targetFramework="net45" />
```
Comment (vguzmanp) [inline LongoMatch.Plugins.GStreamer/packages.config]: Why is everything depending on Mono.Addins now?

File: Tests/Setup.cs [hunk 4]
```diff
@@ -50,7 +50,6 @@ public static void Initialize ()
 			VAS.App.Current = App.Current = new App ();
 			App.InitDependencies ();
 			App.Current.Config = new Config ();
-			App.InitConstants ();
```
Comment (vguzmanp) [inline Tests/Setup.cs]: Why is this no longer needed?

File: Tests/Makefile.am [hunk 5]
```diff
@@ -1,8 +1,7 @@
 PROJECT_FILE = Tests.csproj
 ASSEMBLY=Tests.dll
 
-include $(top_srcdir)/build/build_env.mk
-include $(top_srcdir)/build/build.mk
+include $(top_srcdir)/VAS/build/build.mk
 
 if ENABLE_TESTS
```
Comment (vguzmanp) [inline Tests/Makefile.am]: How are tests going to be managed now? Right now we had:

- VAS.Tests
- Tests -> We had a `Tests` project both in LM and RA, and the assemblies in VAS had a `InternalsVisibleTo[Tests]`, so both could access VAS internals.

In order to keep having internals visible to tests we should have either the same system (`Tests` in longomatch, not in longomatch-core?) or do a similar thing having tests in longomatch that only tests longomatch, and the ones in longomatch-core that only test longomatch-core.

File: tools/ide_osx.sh [hunk 6]
```diff
@@ -1,8 +0,0 @@
-#!/bin/sh
-export CERBERO_PREFIX=/opt/longomatch-build_darwin_darwin_x86_64/dist/darwin_x86_64/
-export PATH=$PATH:/opt/longomatch-build_darwin_darwin_x86_64/build-tools-darwin-x86_64/bin/:$CERBERO_PREFIX/bin/
-export PKG_CONFIG_PATH=$CERBERO_PREFIX/lib/pkgconfig/
-export MONO_PATH=$CERBERO_PREFIX/lib/mono/4.5/:$CERBERO_PREFIX/lib/mono/4.5/Facades
-export DYLD_FALLBACK_LIBRARY_PATH=$CERBERO_PREFIX/lib
-open /Applications/Xamarin\ Studio.app $@ -n
```
Comment (vguzmanp) [inline tools/ide_osx.sh]: Why did you remove(/move) these?

File: Tests/Makefile.am (hunk 5 above)
Comment (vguzmanp) [inline Tests/Makefile.am]: Or is this already working with that?

File: LongoMatch.GUI/LongoMatch.GUI.csproj (hunk 1 above)
Comment (SandraPG85) [inline LongoMatch.GUI/LongoMatch.GUI.csproj]: it should refer the package, this was not properly updated in some previous change

File: LongoMatch.Plugins.GStreamer/packages.config (hunk 3 above)
Comment (SandraPG85) [inline LongoMatch.Plugins.GStreamer/packages.config]: there initial reference was not properly updated, it should refer the nugget.

File: tools/ide_osx.sh (hunk 6 above)
Comment (SandraPG85) [inline tools/ide_osx.sh]: it is not used

Comment (vguzmanp) [inline tools/ide_osx.sh]: These are tools to help in development. They probably should be only in VAS, instead of in every repository. 👌

File: LongoMatch.Plugins.GStreamer/packages.config (hunk 3 above)
Comment (vguzmanp) [inline LongoMatch.Plugins.GStreamer/packages.config]: I don't follow you

Comment (SandraPG85) [inline LongoMatch.Plugins.GStreamer/packages.config]: For what I understood from Andoni at the beginning it was referred as an assembly but later was consumed as a package but it was only updated in the Longomatch.Addins project, so I updated the others.
